src/app/utility/help-panel/help-panel.component.ts

```diff
@@ -10,23 +10,33 @@ import { IdbFacility } from 'src/app/models/idb';
 })
 export class HelpPanelComponent implements OnInit {
 
-
-  selectedFacility: IdbFacility;
+  helpText: string;
+  selectedSource: string;
   selectedFacilitySub: Subscription;
-  constructor(private facilityDbService: FacilitydbService) { }
+  selectedFacilityName: string = 'Facility';
+  
+  constructor(
+    private router: Router,
+    private facilityDbService: FacilitydbService,
+    ) { 
+    router.events.subscribe((route) => {
+      if (route instanceof NavigationEnd) {
+        this.helpText = route.url.replace('/utility/','');
+        this.selectedSource = this.helpText.split('energy-consumption/')[1];
+      }
+    });
+  }
 
   ngOnInit() {
```

Comment:
I'm guessing this was a merge issue. We need the selected facility and save functions for the site to source section